src/app/public/modules/modal/modal.component.scss

```diff
@@ -41,6 +41,10 @@
 .sky-modal-content {
   background-color: #fff;
   z-index: 1;
+
+  &:focus {
+    outline: thin dotted #4d90fe;
```

Comment:
Is this @Blackbaud-ToddRoberts approved?

Reply:
It's how I recreated what he suggested ;p
But I haven't received feedback from him on it yet (and he's out until Monday)

Reply:
I think it looks generally ok; only slight concern is that if this is across browsers, it will look more prominent than other focus in Firefox where the dotted line is black, but if it has to be the same it's not a showstopper

Reply:
Updated to use the browser's color of choice